### PR TITLE
Fix uk (Ukrainian) locale month

### PR DESCRIPTION
**Pull request by Valerika**
Hi! Please, approve my pull request for uk (Ukrainian) locale. Thanks!

### CODE REVIEW

**Comment by iamkun** [comment]
Please fix the unit test.

**Comment by Valerika** [comment]
@iamkun done

File: test/plugin/customParseFormat.test.js [hunk 1]
```diff
@@ -2,10 +2,14 @@ import MockDate from 'mockdate'
...
 import dayjs from '../../src'
 import customParseFormat from '../../src/plugin/customParseFormat'
+import localeData from '../../src/plugin/localeData'
+import localizedFormat from '../../src/plugin/localizedFormat'
 import uk from '../../src/locale/uk'
 import '../../src/locale/zh-cn'
 
 dayjs.extend(customParseFormat)
+dayjs.extend(localizedFormat)
+dayjs.extend(localeData)
```
Comment (iamkun) [inline test/plugin/customParseFormat.test.js:12]: why adding this?

File: src/locale/uk.js [hunk 2]
```diff
@@ -1,6 +1,14 @@
 // Ukrainian [uk]
 import dayjs from 'dayjs'
 
+const monthFormat = 'січня_лютого_березня_квітня_травня_червня_липня_серпня_вересня_жовтня_листопада_грудня'.split('_')
+const monthStandalone = 'січень_лютий_березень_квітень_травень_червень_липень_серпень_вересень_жовтень_листопад_грудень'.split('_')
+
+const monthShortFormat = 'сiч_лют_бер_квiт_трав_черв_лип_серп_вер_жовт_лист_груд'.split('_')
```
Comment (iamkun) [inline src/locale/uk.js:7]: there's no difference in `monthShort`, right?

Comment (Valerika) [reply src/locale/uk.js:7]: yes, you're right

File: test/plugin/customParseFormat.test.js (hunk 1 above)
Comment (Valerika) [reply test/plugin/customParseFormat.test.js]: I'm sorry, just removed

File: src/locale/uk.js (hunk 2 above)
Comment (iamkun) [reply src/locale/uk.js:7]: why not using the original `monthsShort`?

Comment (Valerika) [reply src/locale/uk.js:7]: for the same logic with long names
(I looked Russian locale file)

Comment (iamkun) [reply src/locale/uk.js:7]: There's no need actually.
You can just add `monthFormat ` and `monthStandalone `

Comment (Valerika) [reply src/locale/uk.js:7]: ok

**Comment by iamkun** [comment]
there's one lint error. 


46:15  error  Missing space before value for key 'monthsShort'

**Comment by iamkun** [comment]
Thanks

**Pull request merged**